pkg/network/istio/annotations.go

```diff
@@ -21,4 +21,7 @@ package istio
 
 const (
 	ISTIO_INJECT_ANNOTATION = "sidecar.istio.io/inject"
+
+	// Istio list of virtual interfaces whose inbound traffic (from VM) will be treated as outbound traffic in envoy
+	ISTIO_KUBEVIRT_ANNOTATION = "traffic.sidecar.istio.io/kubevirtInterfaces"
```

Comment:
Although these is another var name in this formatting, please use the standard camel-case format for this var.
We can change the other one later on.

Reply:
I suggest to change them both in a follow-up.
WDYT?

Reply:
While we are at it - we could also drop the `istio` prefix, since it is the same as the package name and thus redundant.

Reply:
Fine by me.

Reply:
@ormergi Please see #12542.